google-cloud-pub-sub-grpc/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/grpc/scaladsl/GooglePubSub.scala

```diff
@@ -69,6 +69,31 @@ object GooglePubSub {
       }
       .mapMaterializedValue(_.flatMap(identity)(ExecutionContexts.sameThreadExecutionContext))
 
+  /**
+   * Create a source that emits messages for a given subscription using a synchronous PullRequest.
+   *
+   * The materialized value can be used to cancel the source.
+   *
+   * @param request the subscription FQRS field is mandatory for the request
+   * @param pollInterval time between PullRequest messages are being sent
+   */
+  def subscribe(
```

Comment:
I believe it would be better to have a specific name and not use overloading. `subscribePull`, `subscribePolling`?

Reply:
@ennru okay then, `subscribeStream` and `subscribePoll`?

Reply:
And what about backward compatibility? Do we want to keep `subscribe` as it is with a deprecation note?

Reply:
No, just pick a specific name for the new method.